packages/main/src/MultiComboBox.js

```diff
@@ -572,17 +579,33 @@ class MultiComboBox extends UI5Element {
 		this._keyDown = false;
 	}
 
-	async _onkeydown(event) {
+	_onkeydown(event) {
+		const isArrowDownCtrl = isDownCtrl(event);
+
 		if (isShow(event) && !this.readonly && !this.disabled) {
 			event.preventDefault();
 			this.togglePopover();
 		}
 
-		if (isUp(event) || isDown(event)) {
-			this._handleArrowNavigation(event);
+		if (isDownShift(event) || isUpShift(event)) {
 			return;
 		}
 
+		if (isUp(event) || isDown(event) || isUpCtrl(event) || isArrowDownCtrl) {
+			this._handleArrowNavigation(event, isArrowDownCtrl);
+			return;
+		}
+
+		// CTRL + Arrow Down navigation is performed by the ItemNavigation module of the List,
+		// here we only implement the text selection of the selected item
+		if (isArrowDownCtrl && !this.allItemsPopover.opened) {
+			setTimeout(() => this._inputDom.setSelectionRange(0, this._inputDom.value.length), 0);
+		}
+
+		if (isLeftCtrl(event) || isRightCtrl(event)) {
```

Comment:
1. On Command + Left, if we have both token and text and the text is selected, for a second you can notice how the focus goes directly to the last token and then we have navigation to the previous page. The expected behaviour is to move the caret to the beginning of the selection In UI5 it works as expected.
2. The same scenario on Arrow Left, the expected behaviour is the same as on Command + Arrow Left, but the actual result is that the focus goes directly to the last token, instead of moving the caret to the beginning.
3. If we are in the List, using only Left or Right Arrow nothing happens as discussed but on Command + Arrow Left/Right we can navigate through the suggestions, which is not expected.  The result is expected to be the same as using only Left/Right Arrow.